apps/wakunode2/external_config.nim

```diff
@@ -30,9 +30,12 @@ type ProtectedTopic* = object
   topic*: string
   key*: secp256k1.SkPublicKey
 
+proc noop() = discard
```

Comment:
?

Reply:
still a wip, need to figure out how we can have a command for wakunode2 which doesn't accept additional args, see my message here - https://discord.com/channels/1110799176264056863/1151319265694199829/1184765875329433681